src/NUnitFramework/framework/Internal/Commands/OneTimeTearDownCommand.cs

```diff
@@ -71,6 +71,9 @@ public override TestResult Execute(TestExecutionContext context)
                 IDisposable disposable = context.TestObject as IDisposable;
                 if (disposable != null && Test is IDisposableFixture)
                     disposable.Dispose();
+
+                //Clear references to test objects to reduce memory usage
+                context.TestObject = null;
```

Comment:
Aha! Do you mean the context is meant to go out of scope? When we first put this patch in, it was necessary to clear the reference on both the fixture and the context. (As mentioned above - this was back on 3.0 though, I'll should check things with the master - that this patch is still working, for one.)